docs/upgrade/2.0-3.0.md

```diff
@@ -144,5 +154,149 @@ server using these tools in `docs/DEPLOY.md`.
 
 ## Suggested upgrade path
 
-* Clone new copy
-* Pull out custom style, locales, etc
+What follows are some steps you can follow to upgrade an existing Madison
+instance from it's current version to the 3.0 version. Because there are some
+major changes to how the project is structured, you will want to take extra care
+to ensure that everything works accordingly.
+
+### Backup current codebase & database
+
+Make sure to create a backup copy of your current codebase, and a dump of your
+current database, should anything go awry.
+
+To copy your codebase, simply running the following command should be enough:
+
+```
+$ cp -r /path/to/madison/codebase /path/to/madison/codebase.backup
+```
+
+To backup your database, assuming you're using MySQL, use the following:
+
+```
+$ mysqldump --user=username --password=password database_name > madison_db_dump.sql
```

Comment:
use `-p` instead of `--password=password` so that our users don't end up with their password in their bash history; `mysqldump` will prompt them.